src/components/Time/DateTimeInput.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import { Tooltip, TextInput, Button, ButtonVariant, Modal } from '@patternfly/react-core';
+import { IntervalInSeconds, TimeInSeconds } from 'types/Common';
+import { toString } from './LocalTime';
+import { KialiIcon, defaultIconStyle } from 'config/KialiIcon';
+import Slider from 'components/IstioWizards/Slider/Slider';
+import { style } from 'typestyle';
+
+interface DateTimeInputProps {
+  minTime: TimeInSeconds;
+  steps?: number;
+  time: TimeInSeconds;
+  tooltip: string;
+
+  onTimeChange: (time: TimeInSeconds) => void;
+}
+
+type DateTimeInputState = {
+  customTime: TimeInSeconds;
+  showCustomTime: boolean;
+};
+
+const defaultSteps: number = 100;
+
+const sliderStyle = style({
+  marginBottom: '2em',
```

Comment:
We have `csx` that we can use for this: `csx.em(2)`, `csx.percent(15)` and so on.

I'm not seeing it being used anywhere else and I have no idea why. @mtho11 do you know why?